package.json

```diff
@@ -1,14 +1,21 @@
 {
   "name": "@first-lego-league/ms-correlation",
-  "version": "0.0.4",
+  "version": "0.0.5",
```

Comment:
I think we can make it first version already. Even though we have made braking changes here, as long as it's on version 0.X npm won't bring us the newer version.